features/api/access.feature

```diff
@@ -0,0 +1,90 @@
+Feature: Subscription status api
+
+  @uses.config.contract_token
+  Scenario Outline: Subscription api when attached/unattached
+    Given a `<release>` `<machine_type>` machine with ubuntu-advantage-tools installed
+    When I attach `contract_token` with sudo
+    When I set the machine token overlay to the following yaml:
+      """
+      machineTokenInfo:
+        contractInfo:
+          id: "TestCId"
+          name: "TestCName"
+          products: []
+        accountInfo:
+          id: "TestAAId"
+          name: "TestAAName"
+        machineId: "TestMId"
+      """
+    When I run `pro api u.pro.access.v1` with sudo
```

Comment:
@dheyay Let's add a non-root check on this test too

Reply:
Updated @lucasmoura